lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -171,10 +171,6 @@ def unstore(customer_vault_id, options = {})
       end
       alias_method :delete, :unstore
 
-      def supports_network_tokenization?
```

Comment:
good question, @lyverovski should we leave this to true on `Braintree Blue` but make it conditional on other `Braintree`?